README.rst

```diff
@@ -28,41 +28,16 @@ not yet available.
 Installation
 ============
 
-Download the zip file and unzip it; or, much better,
-clone the git repository. In either case, change
-your directory to the newly-created ``utide`` distribution
-directory.
-
-.. code:: shell
-
-    python setup.py install
-
-or
-
 .. code:: shell
 
-    python setup.py install --user
-
-if you don't want a system-wide installation.
-
-If you want to work on developing the package, then
-
-.. code:: shell
-
-    python setup.py develop
-
-will allow you to test without reinstalling after making
-a modification.
+    pip install utide
```

Comment:
I guess we should update the PyPI package and the conda package with the latest changes in master.
Are you up to a new release?

Reply:
Absolutely!